File: VB/WindowsFormsApplication662/MySkinTapPainter.vb
Imports System
Imports System.Collections.Generic
Imports System.Drawing
Imports System.Linq
Imports System.Text
Imports DevExpress.XtraTab
Imports DevExpress.XtraTab.Drawing

Namespace WindowsFormsApplication662

    Public Class MySkinTapPainter
        Inherits SkinTabPainter

        Public Sub New(ByVal tabControl As IXtraTab)
            MyBase.New(tabControl)

        End Sub
        Protected Overrides Sub DrawHeaderPage(ByVal e As TabDrawArgs, ByVal row As DevExpress.XtraTab.ViewInfo.BaseTabRowViewInfo, ByVal pInfo As DevExpress.XtraTab.ViewInfo.BaseTabPageViewInfo)
            MyBase.DrawHeaderPage(e, row, pInfo)
            Dim rect As Rectangle = pInfo.Bounds
            rect.Height = 2
            e.Graphics.FillRectangle(Brushes.Red, rect)
            e.Graphics.FillRectangle(New SolidBrush(pInfo.Page.Appearance.Header.BackColor2), rect)
        End Sub
    End Class
End Namespace

File: CS/WindowsFormsApplication662/MySkinTapPainter.cs
using System;
using System.Collections.Generic;
using System.Drawing;
using System.Linq;
using System.Text;
using DevExpress.XtraTab;
using DevExpress.XtraTab.Drawing;

namespace WindowsFormsApplication662 {

    public class MySkinTapPainter : SkinTabPainter {
        public MySkinTapPainter(IXtraTab tabControl)
            : base(tabControl) {

        }
        protected override void DrawHeaderPage(TabDrawArgs e, DevExpress.XtraTab.ViewInfo.BaseTabRowViewInfo row, DevExpress.XtraTab.ViewInfo.BaseTabPageViewInfo pInfo) {
            base.DrawHeaderPage(e, row, pInfo);
            Rectangle rect = pInfo.Bounds;
            rect.Height = 2;
            e.Graphics.FillRectangle(Brushes.Red, rect);
            e.Graphics.FillRectangle(new SolidBrush(pInfo.Page.Appearance.Header.BackColor2), rect);
        }
    }
}
